native/source/mixpanel/detail/mixpanel.cpp

```diff
@@ -55,12 +55,12 @@ namespace mixpanel
     {
         if (worker)
         {
-            throw std::logic_error("Only one Mixpanel instance is allowed at any time");
+            throw std::logic_error("Only one Mixpanel instance at a time is supported.");
         }
-
+        
         if (token.size() < 8)
         {
-            throw std::invalid_argument("You must provide a valid mixpanel token");
+            throw std::invalid_argument("You must provide a valid Mixpanel token");
```

Comment:
Nit: The message on line 58 ends with a `.`; this message probably should too.
I see now that removing the throw-spec on line 43 was a good change for another reason besides readability: this constructor actually throws things that *aren't* `logic_error`! :P  So the throw-spec was already wrong.